post-payments.php

```diff
@@ -191,7 +198,40 @@ public function format_currency( $number ) {
 		return $this->currency_symbol . number_format( $number, 2 );
 	}
 
-	
+	public function download_report() {
+
+		if ( ! empty( $_GET['download'] ) && 'download_report' == $_GET['download'] ) {
+			// restricting to admins only
+			if ( ! current_user_can( 'update_core' ) ) {
```

Comment:
maybe make this capability a property of the class? I can see where someone might want an easy way to adjust it